lib/generators/avo/templates/locales/avo.nb-NO.yml

```diff
@@ -16,6 +16,15 @@ nb-NO:
     resource_updated: 'Ressurs oppdatert'
     resource_created: 'Ressurs generert'
     resource_destroyed: 'Ressurs slettet'
+    record: # HELP NEEDED FOR TRANSLATIONS
+      zero: '%{count} records'
+      one: '%{count} record'
+      other: '%{count} records'
+    destroyed: # HELP NEEDED FOR TRANSLATIONS
+      zero: 'destroyed'
+      one: 'destroyed'
+      other: 'destroyed'
+    resource_batch_destroyed: '%{records} %{destroyed}'
```

Comment:
Can't find nobody to help us on this, want to translate it online or let it like that and wait for someone to translate it?

Reply:
Try and find a similar project that has translations (think Wordpress, e-commerce websites, etc.) or use Google translate. We'll get suggestions from developers if they are wrong.